SPECS/conmon/conmon.spec

```diff
@@ -0,0 +1,399 @@
+%global with_debug 1
+%global with_check 0
```

Comment:
This one's problematic, because it conflicts with the macro we set internally during builds. What happens if you remove this `with_check` line?